src/v/utils/base64.cc

```diff
@@ -110,3 +120,25 @@ ss::sstring iobuf_to_base64(const iobuf& input) {
     output.resize(written);
     return output;
 }
+
+bytes base64url_to_bytes(std::string_view data) {
```

Comment:
nit/dream/just-throwing-the-idea-out-there: it would be cool to validate this method against cryptopp's decoder with a bunch of random data (maybe with the help of a fuzz testing library). I really like Golang's built-in support for fuzz testing, but this might be more work than its benefit in C++.

Reply:
@pgellert we have a few fuzzers in the tree that use clang's fuzzing framework. feel free to use that if whenever you want!

Reply:
Nice, thank you!